src/runners/junitRunner/JUnitRunnerResultAnalyzer.ts

```diff
@@ -98,13 +98,19 @@ export class JUnitRunnerResultAnalyzer extends BaseRunnerResultAnalyzer {
 
     protected getTestId(message: string): string {
         const regexp: RegExp = /\d+,(@AssumptionFailure: |@Ignore: )?(.*?)\((.*?)\)/;
-        const matchResults: RegExpExecArray | null = regexp.exec(message); {
-            if (!matchResults || matchResults.length < 4) {
-                logger.error(`Failed to parse the message: ${message}`);
-                return '';
-            }
+        const matchResults: RegExpExecArray | null = regexp.exec(message);
+        if (matchResults && matchResults.length === 4) {
```

Comment:
This piece of code is not self-explanatory. E.g. What should a valid message look like? What's the meaning of each matched part? 
Suggest to add comments: 
* meaning of the pattern
* a sample message